application/src/main/java/org/togetherjava/tjbot/commands/tags/TagManageCommand.java

```diff
@@ -90,10 +92,18 @@ public TagManageCommand(TagSystem tagSystem) {
     private static void sendSuccessMessage(@NotNull Interaction event, @NotNull String id,
             @NotNull String actionVerb) {
         logger.info("User '{}' {} the tag with id '{}'.", event.getUser().getId(), actionVerb, id);
+        /*
+         * MessageUtils.generateEmbed("Success", "Successfully %s tag '%s'.".formatted(actionVerb,
+         * id), event.getUser(), TagSystem.AMBIENT_COLOR, null)
+         */
+
         event
-            .replyEmbeds(MessageUtils.generateEmbed("Success",
-                    "Successfully %s tag '%s'.".formatted(actionVerb, id), event.getUser(),
-                    TagSystem.AMBIENT_COLOR, null))
+            .replyEmbeds(new EmbedBuilder().setTitle("Success")
+                .setDescription("Successfully %s tag '%s'.".formatted(actionVerb, id))
+                .setAuthor(String.valueOf(event.getUser()))
```

Comment:
Why are you using `setAuthor` now? Thats wrong. Use `setFooter` like it was in the old code. Dont change behavior. Also, `String.valueOf(event.getUser())` is definitely wrong.

Reply:
Now you removed it completely. You have to use `setFooter` in the way it worked before you deleted the method. The username was previously displayed in the footer and you have to replicate that behavior now, otherwise you are changing the behavior.